test/Microsoft.AspNetCore.Razor.Runtime.Test/TagHelpers/TagHelperContextTest.cs

```diff
@@ -23,7 +23,7 @@ public void Constructor_SetsProperties_AsExpected()
 
             // Act
             var context = new TagHelperContext(
-                allAttributes: Enumerable.Empty<IReadOnlyTagHelperAttribute>(),
+                allAttributes: new ReadOnlyTagHelperAttributeList(Enumerable.Empty<TagHelperAttribute>()),
```

Comment:
Huh? `Enumerable.Empty<TagHelperAttribute>()` is a fine `IEnumerable<TagHelperAttribute` and the constructor will do the rest.